### PR TITLE
PR: Give room in nonce checker for new checks.

### LINKED ISSUE #293: Modify nonce checker to give room to new Run now cron link
Required by https://github.com/sudar/email-log-auto-delete-logs/issues/16

### PR DESCRIPTION
**Pull request by esudalaimadan**
Fix #293

Required by https://github.com/sudar/email-log-auto-delete-logs/pull/17
Required by https://github.com/sudar/email-log-summary/pull/7

### CODE REVIEW

**Review by sudar — CHANGES_REQUESTED** [review]
Based on the review comments at https://github.com/sudar/email-log-auto-delete-logs/pull/17 you need to make changes here as well.

File: include/Core/Request/NonceChecker.php [hunk 1]
```diff
@@ -58,15 +58,15 @@ public function check_nonce() {
 				$action = sanitize_text_field( $_REQUEST['action2'] );
 			}
 
-			if ( strpos( $action, 'el-log-list-' ) !== 0 ) {
+			if ( strpos( $action, 'el-' ) !== 0 ) {
 				return;
 			}
 
-			if ( ! isset( $_REQUEST[ LogListPage::LOG_LIST_ACTION_NONCE_FIELD ] ) ) {
+			if ( ! isset( $_REQUEST[ $action . '-nonce-field' ] ) ) {
```
Comment (sudar) [inline include/Core/Request/NonceChecker.php:65]: See https://github.com/sudar/email-log/blob/dev/2.4.3/include/Core/UI/Page/LogListPage.php#L30

The action already has `nonce` in it. So append only `-field` and not `-nonce-field`.

You would require to make the change in the add-on as well.

**Review by sudar — CHANGES_REQUESTED** [review]

**Comment by sudar** [comment]
@esudalaimadan 

I have left a review comment. Also the Style CI checks are failing. Check them as well.

File: include/Core/Request/NonceChecker.php (hunk 1 above)
Comment (esudalaimadan) [reply include/Core/Request/NonceChecker.php:64]: @sudar is going to look into this. So assigning back to him.

**Pull request merged**